specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+        },
+        "x-ms-examples": {
+          "WithdrawPlan": {
+            "$ref": "./examples/WithdrawPlan.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/fetchAllSubscriptionsInTenant": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "#/parameters/SkipTokenParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_FetchAllSubscriptionsInTenant",
+        "description": "Fetch all subscriptions in tenant, only for marketplace admin",
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionsResponse"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "skipToken"
+        },
+        "x-ms-examples": {
+          "FetchAllSubscriptionsInTenant": {
+            "$ref": "./examples/FetchAllSubscriptionsInTenant.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/listNewPlansNotifications": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_ListNewPlansNotifications",
+        "description": "List new plans notifications",
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/NewPlansNotificationsList"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListNewPlansNotifications": {
+            "$ref": "./examples/ListNewPlansNotifications.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/listStopSellOffersPlansNotifications": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_ListStopSellOffersPlansNotifications",
+        "description": "List stop sell notifications for both stop sell offers and stop sell plans",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
```

Comment:
>payload [](http://example.com/codeflow?start=21&length=7)

The name should indicate what the parameter contains, e.g. stopSellSubscriptions.

Reply:
fixed